src/components/question/QuestionList.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Question } from '../../models/Question';
+import Button from '../generic/Button';
+
+const QuestionList = (props: { questions: Question[] }) => {
+  const questions: Question[] = props.questions;
+  return (
+    <>
+      {questions?.map((question, index) => {
+        return (
+          <View key={index} style={styles.buttonView}>
+            <Button
+              finished={getAnsweredQuestion(question)}
+              title={question.title}
+              onButtonPress={() => console.log(`Clicked, ${question.title}`)}
+            />
+          </View>
+        );
+      })}
+    </>
+  );
+};
+
+function getAnsweredQuestion(question: Question): boolean {
+  let answeredQuestion = false;
+  question.questionOptions?.forEach((questionOption) => {
+    questionOption.answers?.forEach((answer) => {
+      answer.answer ? (answeredQuestion = true) : (answeredQuestion = false);
```

Comment:
Wat is het doel van deze regel? Als je nu 1 answer hebt gegeven is die vraag meteen beantwoord terwijl er misschien meerdere antwoorden kunnen zijn? Als een question meer questionOptions heeft krijg je ook een onverwachte output met de huidige loop. 
En kan `answer.answer ? (answeredQuestion = true) : (answeredQuestion = false);` niet gewoon `answeredQuestion  = answer.answer` zijn

Reply:
Ik merkte dat  ik zat te kijken naar alle vragenOpties van onderdeel i.p.v de vraag zelf.  Ik heb het nu wel aangepast en verbeterd in de volgende commit. Was beter het te houden bij answeredQuestion++;